sources/api/thar-be-settings/src/service.rs

```diff
@@ -166,10 +199,24 @@ impl ServiceRestart for model::Service {
 #[cfg(test)]
 mod test {
     use super::*;
-    use maplit::{hashmap, hashset};
+    use maplit::hashmap;
+
+    fn check_equal(a: Option<&Vec<String>>, b: Option<&Vec<String>>) -> bool {
+        let a_value = match a {
+            None => HashSet::new(),
+            Some(a_value) => a_value.iter().collect(),
+        };
+
+        let b_value = match b {
+            None => HashSet::new(),
+            Some(b_value) => b_value.iter().collect(),
+        };
+
+        a_value == b_value
+    }
 
     #[test]
-    fn test_get_affected_service_names() {
+    fn test_get_service_setting_map() {
```

Comment:
nit: Might be just me. But I think this test isn't very clear on what it's testing for. Maybe a few inline comments might help here.